crates/ruma-common/src/identifiers/crypto_algorithms.rs

```diff
@@ -59,6 +59,36 @@ pub enum EventEncryptionAlgorithm {
     _Custom(PrivOwnedStr),
 }
 
+/// A key algorithm to be used to generate a key from a passphrase.
+///
+/// This type can hold an arbitrary string. To check for algorithms that are not available as a
+/// documented variant here, use its string representation, obtained through `.as_str()`.
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, StringEnum)]
+#[non_exhaustive]
+#[cfg_attr(feature = "serde", derive(DeserializeFromCowStr, SerializeAsRefStr))]
+pub enum KeyDerivationAlgorithm {
+    /// PBKDF2
+    #[ruma_enum(rename = "m.pbkdf2")]
+    Pbkfd2,
+    #[doc(hidden)]
+    _Custom(PrivOwnedStr),
```

Comment:
```suggestion
    #[ruma_enum(rename = "m.pbkdf2")]
    Pbkfd2,

    #[doc(hidden)]
    _Custom(PrivOwnedStr),
```

Reply:
FWIW i see `_Custom` hugging the previous item in a lot of other enums, so maybe this should also be added to the style guide?

Reply:
Hm, not sure whether it's worth adding it to the style guide.